cypress.json

```diff
@@ -3,5 +3,6 @@
   "viewportHeight": 1660,
   "projectId": "yt5kwm",
   "defaultCommandTimeout": 8000,
-  "video": false
+  "video": false,
+  "chromeWebSecurity": false
```

Comment:
This is required to be able to access Stripe form in the `iframe`. See https://gist.github.com/mbrochh/460f6d4fce959791c8f947cb30bed6a7